dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
I did a test load and can confirm that the OWL API doesn't appear to recognize these as anonymous.

        Anonymous nodes should most likely already be getting generated for any iri
        with the pattern "host /.well-known/genid/" as they could already be occurring in the wild.

Perhaps this is something we can do in SciGraph, @jnguyenx?